libs/market-depth/src/lib/orderbook.spec.tsx

```diff
@@ -183,4 +183,53 @@ describe('Orderbook', () => {
     );
     expect(result.getByTestId('scroll').scrollTop).toBe(6 * rowHeight);
   });
+
+  it('should format correctly the numbers on resolution change', async () => {
+    const onClickSpy = jest.fn();
+    window.innerHeight = 11 * rowHeight;
+    const result = render(
+      <Orderbook
+        decimalPlaces={decimalPlaces}
+        positionDecimalPlaces={0}
+        onClick={onClickSpy}
+        fillGaps
+        {...generateMockData(params)}
+        onResolutionChange={onResolutionChange}
+      />
+    );
+    await waitFor(() => screen.getByTestId(`bid-vol-${params.midPrice}`));
+    const scrollElement = result.getByTestId('scroll');
+
+    fireEvent.click(result.getByTestId('price-122934'));
+    expect(onClickSpy).toBeCalledWith('122.934');
+
+    expect(scrollElement.scrollTop).toBe(91 * rowHeight);
```

Comment:
question (non blocking): Do we need all the test code relating to scroll position? They seem like they are unrelated to this test

Reply:
removed